corehq/apps/reports/util.py

```diff
@@ -602,3 +602,21 @@ def _add_new_user_to_HQ_group(session, local_user):
         _sync_tableau_users_with_hq(domain)
         # Sync the TableauUser model with Tableau users on the remote Tableau instance
         _sync_tableau_users_with_remote(domain)
+
+
+# Attaches to the parse_web_users method
+def add_on_tableau_details(domain, web_user_dicts):
+    tableau_users_for_domain = TableauUser.objects.filter(server=TableauServer.objects.get(domain=domain))
+    for web_user_dict in web_user_dicts:
+        if not web_user_dict['last_login (read only)'] == 'N/A':
+            tableau_user = tableau_users_for_domain.get(username=web_user_dict['username'])
+            web_user_dict['tableau_role'] = tableau_user.role
+    return web_user_dicts
+
+
+# Attaches to the create_or_update_web_users method
+def import_tableau_user(domain, user, remove, role):
+    if remove:
```

Comment:
nit: I'm curious if it makes more intuitive sense with how the rest of the `create_or_update_web_users` method is laid out to group this check in the the `if remove` check down below on line 776. This would mean checking the FF variable twice, but would save an extra param being passed in here.